src/main/java/com/selfxdsd/selfweb/api/ContributorApi.java

```diff
@@ -220,8 +220,9 @@ public ContributorApi(final User user) {
         @PathVariable final String owner,
         @PathVariable final String name,
         @PathVariable final int invoiceId,
-        @RequestParam("role") final String role
-    ) throws IOException {
+        @RequestParam("role") final String role,
+        @RequestParam(value = "view", required = false) final boolean view
```

Comment:
@criske Why is this parameter needed? :D 

There are a bit more changes than I expected.